data/build.gradle

```diff
@@ -116,10 +116,10 @@ configurations {
   all*.exclude module: 'protobuf-java'
 }
 
-tasks.withType(Test) {
+ tasks.withType(Test) {
   jacoco.includeNoLocationClasses = true
```

Comment:
we will have to add `jacoco.excludes = ['jdk.internal.*']` in all the build.gradle files.

Reply:
This won't make a difference now but i will still add it up